status-client-spec.md

```diff
@@ -156,14 +156,14 @@ Status nodes that want to be discovered MUST register to Discovery V5 and/or Ren
 with the `whisper` topic. Status nodes that are mail servers and want to
 be discoverable MUST additionally register with the `whispermail` topic.
 
-The recommended strategy is to use both mechanisms but at the same time implement a structure
+It is RECOMMENDED to use both mechanisms but at the same time implement a structure
```

Comment:
Is this an official term as per RFC?

Reply:
> 3. SHOULD   This word, or the adjective "RECOMMENDED", mean that there
   may exist valid reasons in particular circumstances to ignore a
   particular item, but the full implications must be understood and
   carefully weighed before choosing a different course.